recipe/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "k-means-constrained" %}
+{% set version = "0.7.2" %}
```

Comment:
```suggestion
{% set version = "0.5.1" %}
```

Reply:
Sticking with 0.7.2. Thanks for pointing that out.